src/components/app-header/app-header.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import './app-header.css';
+import PropTypes from "prop-types";
+
+const AppHeader = ({toDo, done}) => 
+  toDo && done ? (
```

Comment:
I suggest using "if else" for easy reading of code.
`if (!toDo && !done) {
  return null;
}`